CONTRIBUTING.md

```diff
@@ -123,9 +124,14 @@ To test your changes locally, you can run the project from IntelliJ or gradle.
   ```
 
 
+## Guidelines
+
+- AWS Explorer should not have "dependencies" (such as `sam` or `cloud-debug`). It should work without needing to install extra stuff.
```

Comment:
There's probably a lot more we can put into this area - thanks for getting it started!

Reply:
Yeah, we may want to move it to the `designs/` folder later, but it's good to have a place to write things down meanwhile.